src/steamship/plugin/service.py

```diff
@@ -100,15 +100,15 @@ def model_cls(self) -> Type[TrainableModel]:
 
     def run(self, request: PluginRequest[T]) -> Union[U, Response[U]]:
         """Loads the trainable model before passing the request to the `run_with_model` handler on the subclass."""
-        self.logger.info("TrainablePluginService:run() - Loading model")
+        logging.info("TrainablePluginService:run() - Loading model")
```

Comment:
I love how detailed your logging statement is, this makes debugging a breeze! I am afraid that we won't always be so detailed. 

Would it be an option to upgrade the logging format to include part of this automatically? 

An example:
```
FORMAT = "[%(filename)s:%(lineno)s - %(funcName)s] %(message)s"
```

Reply:
At least for remote logging, some of these fields are already in the structured part of the message (which makes it to structured fields in Elastic).  I'll add filename and lineno!